mods/cnc/chrome/settings-display.yaml

```diff
@@ -160,14 +160,23 @@ Container@DISPLAY_PANEL:
 					Height: 20
 					Children:
 						Container@UI_FEEDBACK_CHECKBOX_CONTAINER:
+							X: 10
+							Width: PARENT_RIGHT / 2 - 10
+							Children:
+								Checkbox@UI_FEEDBACK_CHECKBOX:
+									Width: PARENT_RIGHT
+									Height: 20
+									Font: Regular
+									Text: Show UI Feedback Notifications
+						Container@TRANSIENTS_CHECKBOX_CONTAINER:
 							X: PARENT_RIGHT / 2 + 10
 							Width: PARENT_RIGHT / 2 - 20
 							Children:
-								Checkbox@UI_FEEDBACK_CHECKBOX:
+								Checkbox@TRANSIENTS_CHECKBOX:
 									Width: PARENT_RIGHT
 									Height: 20
 									Font: Regular
-									Text: UI Feedback in Transients Panel
+									Text: Show Transient Notifications
```

Comment:
I'm not sure the average user will understand what Transient Notifications are. I didn't anyways.
Perhaps 'Show Speech Notifications as Text'?

Reply:
I think a tooltip would do the job.

Reply:
![image](https://user-images.githubusercontent.com/1355810/155129932-25447460-1044-45ab-93d6-b9265ba60ab3.png)

So something like this?

Reply:
> So something like this?

I think that covers it.